infra/main.bicep

```diff
@@ -30,6 +30,16 @@ param hostingPlanName string = 'hosting-plan-${resourceToken}'
 ])
 param hostingPlanSku string = 'B3'
 
+@description('The sku tier for the App Service plan')
+@allowed([
+  'Basic'
```

Comment:
Thanks for this, does Free and Shared need to be added to this list for the `F1` and `D1` skus?

Reply:
Good callout. Added.